
# Tensor3D.msg
std_msgs/Float32MultiArray data
std_msgs/Int32MultiArray shape